leveldb/leveldb.go

```diff
@@ -0,0 +1,101 @@
+package leveldb
```

Comment:
The name of this package was duplicated to **github.com/syndtr/goleveldb/leveldb** please consider another name for this wrap up like: lvldb